src/Licenses/Entities/License/Prediction.php

```diff
@@ -0,0 +1,107 @@
+<?php
+
+namespace ArrowSphere\PublicApiClient\Licenses\Entities\License;
+
+use ArrowSphere\PublicApiClient\AbstractEntity;
+use ArrowSphere\PublicApiClient\Exception\EntityValidationException;
+
+/**
+ * Class ActiveSeats
+ */
+class Prediction extends AbstractEntity
+{
+    public const COLUMN_CURRENCY = 'currency';
+    public const COLUMN_UPDATE_DATE_OF_ML_SCRIPT = 'updatedAt';
```

Comment:
I don't understand this constant name
```suggestion
    public const COLUMN_UPDATED_AT = 'updatedAt';
```